main.c

```diff
@@ -530,6 +617,10 @@ int main(int argc, const char *argv[])
 	wtype.keyboard = zwp_virtual_keyboard_manager_v1_create_virtual_keyboard(
 		wtype.manager, wtype.seat
 	);
+	// TODO: make this optional
```

Comment:
Should I handle the case of the virtual pointer protocol not being available? or should I assume it is there?